js/src/forum/components/UserCard.js

```diff
@@ -30,7 +30,7 @@ export default class UserCard extends Component {
     const badges = user.badges().toArray();
 
     return (
-      <div className={'UserCard ' + (this.attrs.className || '')} style={color ? { backgroundColor: color } : ''}>
+      <div className={'UserCard ' + (this.attrs.className || '')} style={color ? { '--usercard-bg': color } : ''}>
```

Comment:
This is a more common pattern used in JSX. I know it was here before, but we might as well use it.

```suggestion
      <div className={'UserCard ' + (this.attrs.className || '')} style={color && { '--usercard-bg': color }}>
```